content/community/governance/index.md

```diff
@@ -51,9 +56,9 @@ The [Project Management Office (PMO)](/community/pmo) supports the different bod
 
 **Proposed governance workflow:**
 
-* [Galaxy Exec](/community/exec/): group of PIs including Frederik Coppens, Björn Grüning, Ross Lazarus, Andrew Lonie, Anton Nekrutenko, and Michael Schatz.
-* [Global Steering Committee](/community/steering/): collaborators from active Galaxy communities.
-* [Working groups](/community/wg/): groups of community members responsible for various parts of the Galaxy ecosystem.  
+* [Galaxy Executive Board](/community/governance/geb): group of PIs including Frederik Coppens, Björn Grüning, Ross Lazarus, Andrew Lonie, Anton Nekrutenko, and Michael Schatz.
+* [Galaxy Community Board](/community/governance/gcb): collaborators from active Galaxy communities.
+* [Galaxy Technical Board](/community/governance/gtb): groups of community members responsible for various parts of the Galaxy ecosystem.
```

Comment:
```suggestion
* [Galaxy Technical Board](/community/governance/gtb): representatives from Galaxy working groups.
```

Reply:
representatives, or possibly leads/heads/chairs? How does this work in practice?